### PR TITLE
Fixes #1133:  Change /explore HTML title to "Explore"

### LINKED ISSUE #1133: Change /explore HTML title to "Explore"
Title in the tab still says "Gallery" on webmaker.org/explore. It should be "Explore".

screenshot 2015-01-19 16 30 06

### PR DESCRIPTION
**Pull request by gesa**

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {% include "macros.html" %}
 
-{% block title %}{{ gettext("Gallery") }}{% endblock %}
```
Comment (secretrobotron) [inline views/index.html]: Fails `grunt validate` because `Gallery` is still in the locale strings. Can probably just remove it.

**Comment by secretrobotron** [comment]
R-
Just fix up that `Gallery` string. `grunt validate` should be clean.

**Comment by secretrobotron** [comment]
p.s. @alicoding cleaned up some old strings here: #1194

**Comment by gesa** [comment]
Grunt validate is clean now, merging!

**Pull request merged**